src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -1213,6 +1215,57 @@ public void readonly() {
     sendCommand(READONLY);
   }
 
+  public void geoadd(byte[] key, double longitude, double latitude, byte[] member) {
+    sendCommand(GEOADD, key, toByteArray(longitude), toByteArray(latitude), member);
+  }
+
+  public void geoadd(byte[] key, Map<byte[], GeoCoordinate> memberCoordinateMap) {
```

Comment:
@HeartSaVioR question: Is there a specific reason you decided to use a `Map<byte, Geocoordinate>` instead of just making it a `Set<GeoCoordinate>` and include the member inside the GeoCoordinate bean?. Reading the `geoadd` command documentation it says the redis stores the coordinates as a sorted set internally. That doesn't mean Jedis should do the same thing, but I was just thinking what would be the difference between using a Map or Set for us.